example/lib/main.dart

```diff
@@ -76,7 +77,8 @@ class MainContent extends StatelessWidget {
         enhancedKycAsyncButton(),
         documentVerificationButton(context),
         smartSelfieEnrollmentButton(context),
-        smartSelfieAuthenticationButton(context)
+        smartSelfieAuthenticationButton(context),
+        biometricKYCButton(context)
```

Comment:
Let's match the style used above with `enhancedKyc`
```suggestion
        biometricKycButton(context)
```

Reply:
Done